UIKit_HM/UIKit_HM/ViewControllers/TextViewController.swift

```diff
@@ -0,0 +1,245 @@
+//  ViewController.swift
+//  UIKit_HM
+//  Created by Мажит Закиров on 21.06.2021.
+
+import UIKit
+
+final class TextViewController: UIViewController {
+    
+    // MARK: - Visial Companents
+    
+    private let mainLabel = UILabel()
+    
+    private let fontWrapperView = UIView()
+    private let fontLabel = UILabel()
+    private let fontSlider = UISlider()
+    private let fontSliderValueLabel = UILabel()
+    
+    private let shadowWrapperView = UIView()
+    private let shadowLabel = UILabel()
+    private let shadowSlider = UISlider()
+    private let shadowSliderValueLabel = UILabel()
+    
+    private let shadowColorWrapperView = UIView()
+    private let shadowColorLabel = UILabel()
+    private let shadowColorSlider = UISlider()
+    private let shadowColorValueLabel = UILabel()
+    
+    // MARK: - Life Circle
```

Comment:
```suggestion
    // MARK: - Life Cycle
```
круг жизни это не совсем то, что надо